.gitignore

```diff
@@ -54,3 +54,6 @@ README.html
 .mypy_cache
 
 !tests/.env
+
+#IDEs
+.idea/
```

Comment:
```suggestion
```

This belongs in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
Huh, actually didn't know that was a thing